static/js/components/LearnerSearch.js

```diff
@@ -132,7 +132,7 @@ export default class LearnerSearch extends SearchkitComponent {
   countryNameTranslations: Object = makeCountryNameTranslations();
   degreeTranslations: Object = makeDegreeTranslations();
 
-  getNumberOfCoursesInProgram = (): number => {
+  getNumberOfCoursesInProgram: Function = (): number => {
```

Comment:
The `: Function` annotation here (and on line 147 below) isn't necessary any longer (I actually removed it here in a different PR earlier today 😆 )